### PR TITLE
Update streams

### LINKED ISSUE #16487: Broken: LNTV3
### Broken Link

https://edge1.laotv.la/live/LNTV3/index.m3u8

### What happened to the stream?

Not loading

### Notes (optional)

The Stream URL for Unitel's laotv has changed

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### LINKED ISSUE #17424: Broken: Vision Latina Panamá
### Broken Link

https://5940924978228.streamlock.net/8095/8095/playlist.m3u8

### What happened to the stream?

Not loading

### Notes (optional)

The channel can be viewed without problems on its website https://visionlatinapanama.com/

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### LINKED ISSUE #17479: Broken: NASA TV Media (720p)
### Broken Link

https://ntv2.akamaized.net/hls/live/2013923/NASA-NTV2-HLS/master.m3u8

### What happened to the stream?

Not loading

### Notes (optional)

https://raw.githubusercontent.com/iptv-org/iptv/master/streams/us.m3u

Stream is no longer active currently displaying this:

Screenshot_20240911_222444

### Contributing Guide

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by Alstruit**
Cleanup on broken links.

Closes #16487, Closes #17424, Closes #17479

### CODE REVIEW

File: streams/us.m3u [hunk 1]
```diff
@@ -135,12 +135,6 @@ https://video1.getstreamhosting.com:1936/8112/8112/playlist.m3u8
 https://cdn3.wowza.com/5/ZWQ1K2NYTmpFbGsr/BEK-WOWZA-1/smil:BEKPRIMEeast.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="BEKSportsWest.us",BEK TV Sports West (720p)
 https://cdn3.wowza.com/5/ZWQ1K2NYTmpFbGsr/BEK-WOWZA-1/smil:BEKPRIMEW.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="BetterHealthTV.us",Better Health TV (480p)
-https://tgn.bozztv.com/betterlife/betterhealth/betterhealth/index.m3u8
```
Comment (Dum4G) [inline streams/us.m3u:139]: image

File: streams/us.m3u [hunk 2]
```diff
@@ -135,12 +135,6 @@ https://video1.getstreamhosting.com:1936/8112/8112/playlist.m3u8
...
-#EXTINF:-1 tvg-id="BetterLifeNatureChannel.us",Better Life Nature Channel (480p)
-https://tgn.bozztv.com/betterlife/betternature/betternature/index.m3u8
```
Comment (Dum4G) [inline streams/us.m3u:141]: image

File: streams/us.m3u [hunk 3]
```diff
@@ -135,12 +135,6 @@ https://video1.getstreamhosting.com:1936/8112/8112/playlist.m3u8
...
-#EXTINF:-1 tvg-id="BetterLifeTV.us",Better Life TV (720p)
-https://tgn.bozztv.com/betterlife/betterlife/betterlife/index.m3u8
```
Comment (Dum4G) [inline streams/us.m3u:143]: image

File: streams/us.m3u [hunk 4]
```diff
@@ -491,8 +485,6 @@ https://livefta.malimarcdn.com/ftaedge00/laothaius.sdp/playlist.m3u8
 https://cdn.streamingcpanel.com:3784/live/latinzonetvlive.m3u8
 #EXTINF:-1 tvg-id="LawCrime.us",Law & Crime (720p)
 https://lawcrime.s.llnwi.net/h72/lawcrimech2/playlist_scte.m3u8
-#EXTINF:-1 tvg-id="LifevisionTV.us",LifevisionTV (406p) [Not 24/7]
-https://tgn.bozztv.com/lifevision/lifevision/index.m3u8
```
Comment (Dum4G) [inline streams/us.m3u:495]: image

File: streams/us.m3u [hunk 5]
```diff
@@ -565,16 +557,6 @@ https://sydandmacky.vju.tv/sydandmacky/main.m3u8?test=1
 https://mtchls.wns.live/hls/stream.m3u8
 #EXTINF:-1 tvg-id="NagoTV.us",Nago TV (240p)
 https://lakay.online/public/nagotv/index.fmp4.m3u8
-#EXTINF:-1 tvg-id="NASATV.us",NASA TV (720p)
-http://iphone-streaming.ustream.tv/uhls/6540154/streams/live/iphone/playlist.m3u8
-#EXTINF:-1 tvg-id="NASATVISSViews.us",NASA TV ISS Views (480p) [Not 24/7]
-http://iphone-streaming.ustream.tv/uhls/9408562/streams/live/iphone/playlist.m3u8
-#EXTINF:-1 tvg-id="NASATVMedia.us",NASA TV Media (720p)
-https://ntv2.akamaized.net/hls/live/2013923/NASA-NTV2-HLS/master.m3u8
```
Comment (Dum4G) [inline streams/us.m3u:573]: image
valid

File: streams/xk.m3u [hunk 6]
```diff
@@ -13,6 +13,8 @@ https://ssh101.bozztv.com/ssh101/rtvarberia4/playlist.m3u8
 https://ssh101.bozztv.com/ssh101/rtvarberia5/playlist.m3u8
 #EXTINF:-1 tvg-id="TVArberiaRetroHits.xk",TV Arbëria Retro Hits (720p)
 https://ssh101.bozztv.com/ssh101/rtvarberiaretro/playlist.m3u8
+#EXTINF:-1 tvg-id="TVArta.xk",TV Arta (1080p)
+https://ub1doy938d.gjirafa.net/live/Gfsqdsr7FewrYClU3ACEGZvCHktt2wse/yt1q1x.m3u8
```
Comment (Dum4G) [inline streams/xk.m3u:17]: image

**Review by Dum4G — CHANGES_REQUESTED** [review]

**Review by Dum4G — APPROVED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**